app/assets/stylesheets/home.scss

```diff
@@ -6,3 +6,7 @@
   margin-top: 30px;
   font-size: 22px;
 }
+.navigation-links{
+  list-style:none;
```

Comment:
Properties should be ordered font-size, list-style<br>Colon after property should be followed by one space